overrides/android-override.json

```diff
@@ -113,6 +113,10 @@
                     {
                         "domain": "xfinity.com",
                         "reason": "https://github.com/duckduckgo/privacy-configuration/issues/1075"
+                    },
+                    {
+                        "domain": "hulu.com",
```

Comment:
I think we can remove from `omitApplicationSites`, I don't think `Omit Webview Version` tests the interaction between the two

Reply:
Do we know that?